### PR TITLE
switch to use the latest systemjs, transpile files in browser, add

### PR DESCRIPTION
**Pull request by vladima**
node_modules to gitignore file

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,11 +0,0 @@
-{
-    "compilerOptions": {
-        "emitDecoratorMetadata": true,
-        "module": "amd",
```
Comment (mhegazy) [inline tsconfig.json:4]: you still need this for the editing experience

File: tsd.json [hunk 2]
```diff
@@ -1,12 +0,0 @@
-{
-  "version": "v4",
-  "repo": "borisyankov/DefinitelyTyped",
-  "ref": "master",
-  "path": "typings",
-  "bundle": "typings/tsd.d.ts",
-  "installed": {
-    "angular2/angular2.d.ts": {
```
Comment (mhegazy) [inline tsd.json:8]: you still need this for the editors/IDEs to be able to give intellisense.

File: readme.md [hunk 3]
```diff
@@ -1,14 +1,6 @@
 **install:**
 ```
-npm install tsd
-tsd reinstall
```
Comment (mhegazy) [inline readme.md:4]: i think you still need this as well.

Comment (vladima) [inline readme.md:4]: fair enough, moved it to a separate section

**Pull request merged**